servers/visual_server.h

```diff
@@ -263,8 +267,9 @@ class VisualServer : public Object {
 		ARRAY_FLAG_USE_2D_VERTICES = ARRAY_COMPRESS_INDEX << 1,
 		ARRAY_FLAG_USE_16_BIT_BONES = ARRAY_COMPRESS_INDEX << 2,
 		ARRAY_FLAG_USE_DYNAMIC_UPDATE = ARRAY_COMPRESS_INDEX << 3,
+		ARRAY_FLAG_USE_OCTAHEDRAL_COMPRESSION = ARRAY_COMPRESS_INDEX << 4,
```

Comment:
This should be added to `_bind_method()` in the `.cpp` (see other constants, and make sure to keep the order).

Reply:
Ahh this is what I was missing, because I had referenced the new constant before in another commit but doctool was not recognizing the new constant, so I removed it, I didn't realize there was a `_bind_method()` I need to call for doctool to recognize it, thanks for the pointer!